text/0001-pando-cli.md

```diff
@@ -0,0 +1,211 @@
+will be used to interact with projects that use Pando. The CLI will be written in
+Node.js and will be published to NPM.
+
+# Basic example
+
+```bash
+pnpx pando setup
+```
+
+# Motivation
+
+## Why are we doing this?
+
+As Pando grows, we need a way to ensure that all projects that use Pando are setup
+correctly in a way that requires minimal effort from the developer. This CLI will
+be the primary way to interact with Pando.
+
+## What use cases does it support?
+
+- Setup a project to use Pando
+- Update a project to use the latest version of Pando
+
+## What is the expected outcome?
+
+The outcome should be a CLI that makes it easy to setup and update projects that
+use Pando. This means it should ultimately relieve the developer from having to
+manually configure/update their project to use Pando.
+
+The CLI should also be a delightful experience to use. This means that it should
+be easy to use and should provide helpful feedback when something goes wrong. The
+feedback should be both visual and written (colors, emojis, animations, etc.).
+
+# Detailed design
+
+## CLI commands
+
+### `setup`
+
+The `setup` command will be used to setup a project to use Pando. It should do the
+following:
+
+#### 1. Welcome the user to Pando CLI
+
+This should be a delightful welcome message that includes emoji visuals at the least.
+
+#### 2. Install required dependencies
+
+The CLI should install the following dependencies based on local PM of choice (bun, npm, yarn, pnpm):
+
+- `@pluralsight/{react,icons}`
+
+#### 3. Install required devDependencies
+
+The CLI should install the following dependencies based on local PM of choice (bun, npm, yarn, pnpm):
+
+- `@pluralsight/panda-preset`
+- `@pandacss/dev`
+- `postcss` (if not already installed)
+- `autoprefixer` (if not already installed)
+
+#### 4. Setup Panda & PostCSS
+
+The CLI should follow the steps outlined in the [Panda documentation](https://panda-css.com/docs/installation/postcss) for using PostCSS.
+
+Additionally, the CLI should add `autoprefixer` to the postcss config file.
+
+#### 5. Ask the user some Panda configuration questions
+
+The CLI should ask the user the following questions to help fill in any remaining Panda configuration gaps needed:
+
+- **What files would you like to include?** This should use a placeholder that matches step 3 in the PostCSS setup documentation.
+- **What files would you like to exclude?** This should use a placeholder that matches step 3 in the PostCSS setup documentation.
+- **What framework would you like to use?** This should use a placeholder of `react`. [Available Options](https://panda-css.com/docs/references/config#jsxframework)
+
+The config file should include the comments that using `panda init` generally provides.
+
+#### 6. Create `pando.entry.css` file
+
+The CLI should create a `pando.entry.css` file in the `src` directory of the project. This file should be the main CSS file for the project.
```

Comment:
Right now it's a design preference because the differences I've seen in entry CSS files on each team is so dramatic it's a little mind blowing. This is just the safe/scalable way that accomplishes a few things:

1. Provides an easy to understand CSS entry for Pando within projects
2. Allow the CLI to scale without risking code breaking changes to existing projects
3. Easy to add into a project via JS import to the main
4. Easy to remove when needed without loosing history (i.e. just remove the import and keep the file)

Reply:
Great question though.